src/QueryBusInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace ZfrCommandBus;
+
+/**
+ * @author Daniel Gimenes
+ */
+interface QueryBusInterface extends MessageBusInterface
+{
+    /**
+     * @param object $query
+     *
+     * @return array
+     */
+    public function dispatch($query): array;
```

Comment:
We can typehint on object on PHP 7.1 no?

Reply:
Nope :/